site/docs/guides/responsive.md

```diff
@@ -0,0 +1,273 @@
+    </ThemeProvider>
+  );
+}
+```
+On the browser this looks like this:
+<Image img="tutorial/responsive-01.jpg" />
+
+Okay, so we are not winning any design awards, but they don't pay me enough for that (or at all). 
+To change the width of the frame container, one approach would be to wrap it in a div and change the width of that div:
+
+```tsx
+          <div
+            className=""
+            style={{ width: breakpoints[breakpoint], margin: '0 auto' }}
+          >
+            <Frame>
+              <Element canvas is={Container}>
+                    <div className="sm:text-red md:text-white 2xl:text-black">
+                      This text is red on mobile, white on tablets & black on
+                      desktops
+                    </div>
+              </Element>
+            </Frame>
+          </div>
+```
+
+This actually works, well, sort of:
+<Image img="tutorial/responsive-02.gif" />
+
+As you can see, we have a problem. The change is not changing color!! If you think about it, it makes sense, we are not changing the size of the screen, only of the component so our media queries will not trigger. 
+
+At this point we have two options:
+1. Have each component keep a state of the classes it should apply based on each breakpoint and pass around the current breakpoint via something like a store or a context. I would not recommend this personally.
+2. Use an iframe which is how all the big website design apps do it
+
+### Using an Iframe
+Before you try to replace our div with an iframe and call it a day, I tried, it doesn't work :( We need an iframe-like thing that works well with react. There are probably other options, I encounrage you to do your own research but one I have used and works is the `react-frame-component` package.
+
+```bash
+# lets install it
+yarn add react-frame-component
+```
+
+Then we can wrap our first frame from craftjs with the frame from `react-frame-component` and get something like this:
+
+```tsx
+          <div
+            className=""
+            style={{ width: breakpoints[breakpoint], margin: '0 auto' }}
+          >
+            <IFrame className="block h-full w-full">
+              <Frame>
+                <Element canvas is={Container}>
+                    <div className="sm:text-red md:text-white 2xl:text-black">
+                      This text is red on mobile, white on tablets & black on
+                      desktops
+                    </div>
+                </Element>
+              </Frame>              
+            </IFrame>
+          </div>
+```
+
+> **NOTE: if using SSR framework like next**
+> You must ensure that this component runs only on the client
+
+If you did this correctly you should be able you should see the text on the screen and be able to resize the iframe correctly. **But, and this is important** the text will still not change color. So, we are back where we started, see? progress. This is because of the [Iframe CSS Problem](#the-iframe-css-problem)
```

Comment:
Hi! First of all, props for the great addition!
Second, I think you might have here a mistake `... you should be able you should see...`

Reply:
You are totally right. My bad. Will fix this & push the 2nd part of the guide once I have finished my exams in 2 weeks. Thanks for catching it!